src/Seconds.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace IconLanguageServices\Time;
+
+final class Seconds implements ConvertsToSeconds

```

Comment:
Any reason these are all final?

Reply:
This library will be public unless we pay for Github, so I'm just trying to limit the scope for use errors.